devnet/README.md

```diff
@@ -1,25 +1,24 @@
-# /!\ Install Nix + Flakes
+# Overview
 
-1. https://nixos.org/download.html
-2. https://nixos.wiki/wiki/Flakes
+Devnet configs based on [nix](../docs/nix.md)
 
-# Run locally
+## Run locally
 
-1. `./update.sh REVISION` where `REVISION` is the latest deployed commit hash.
-2. Go back to the root directory and run `nix develop .#devnet` then run the devnet using `run-dali-dev`.
+1. `./update.sh REVISION` where `REVISION` is the latest deployed git commit hash.
+2. Go back to the root directory and run `nix develop .#sre` then run the devnet using `run-dali-dev`.
 3. Reach alice at `https://polkadot.js.org/apps/?rpc=ws://localhost:9944#/explorer`
 
-# GCE
+## GCE
 
-/!\ Download your GCE service account key and save it as `ops.json`.
+Download your GCE service account key and save it as `ops.json`.
 
-## Deploy
+### Deploy
 
-1. `nix develop .#deploy`
+1. `nix develop .#sre`
```

Comment:
What does .#sre mean in this context? The environment used by SREs? Can we rename to something more descriptive?

Reply:
shells are aligned with teams.

there is sre team => there is sre shell
there is developers => 
xcvm-developer => 

etc
writers, testers

Reply:
devnet is deploy from #sre shell

Reply:
will documente that nix eliminates difference of ci vs not if you have enough credentials, so until there will be need for optimization, ci commands can stick with team shells. so once sre team get running something locally, it will run remotely too.